mne/inverse_sparse/tests/test_mxne_inverse.py

```diff
@@ -334,3 +338,78 @@ def test_split_gof_meg(forward, idx, weights):
     want = 100 * want / want.sum()
     assert_allclose(gof_split, want, atol=1e-3, rtol=1e-2)
     assert_allclose(gof_split.sum(), 100, rtol=1e-5)
+
+
+@pytest.mark.parametrize('n_sensors, n_dipoles, n_times', [
+    (10, 15, 7),
+    (20, 60, 20),
+])
+@pytest.mark.parametrize('nnz', [2, 4])
+@pytest.mark.parametrize('corr', [0.75])
+@pytest.mark.parametrize('n_orient', [1, 3])
+def test_mxne_inverse_sure_synthetic(n_sensors, n_dipoles, n_times, nnz, corr,
+                                     n_orient, snr=4):
+    """Tests SURE criterion for automatic alpha selection on synthetic data."""
+    rng = np.random.RandomState(0)
+    sigma = np.sqrt(1 - corr ** 2)
+    U = rng.randn(n_sensors)
+    # generate gain matrix
+    G = np.empty([n_sensors, n_dipoles], order='F')
+    G[:, :n_orient] = np.expand_dims(U, axis=-1)
+    n_dip_per_pos = n_dipoles // n_orient
+    for j in range(1, n_dip_per_pos):
+        U *= corr
+        U += sigma * rng.randn(n_sensors)
+        G[:, j * n_orient:(j + 1) * n_orient] = np.expand_dims(U, axis=-1)
+    # generate coefficient matrix
+    support = rng.choice(n_dip_per_pos, nnz, replace=False)
+    X = np.zeros((n_dipoles, n_times))
+    for k in support:
+        X[k * n_orient:(k + 1) * n_orient, :] = rng.normal(
+            size=(n_orient, n_times))
+    # generate measurement matrix
+    M = G @ X
+    noise = rng.randn(n_sensors, n_times)
+    sigma = 1 / np.linalg.norm(noise) * np.linalg.norm(M) / snr
+    M += sigma * noise
+    # inverse modeling with sure
+    alpha_max = norm_l2inf(np.dot(G.T, M), n_orient, copy=False)
+    alpha_grid = np.geomspace(alpha_max, alpha_max / 10, num=15)
+    _, active_set, _ = _compute_mxne_sure(M, G, alpha_grid, sigma=sigma,
+                                          n_mxne_iter=5, maxit=3000, tol=1e-4,
+                                          n_orient=n_orient,
+                                          active_set_size=10, debias=True,
+                                          solver="auto", dgap_freq=10,
+                                          random_state=0, verbose=False)
+    assert_equal(np.count_nonzero(active_set, axis=-1), n_orient * nnz)
+
+
+def test_mxne_inverse_sure():
```

Comment:
you should decorate this test saying that it requires testing data. See how done elsewhere